api/v1beta1/hyperconverged_types.go

```diff
@@ -450,6 +450,14 @@ type HyperConvergedFeatureGates struct {
 	// +kubebuilder:default=false
 	// +default=false
 	AutoResourceLimits *bool `json:"autoResourceLimits,omitempty"`
+
+	// Enable KubeVirt to request up to two additional dedicated CPUs
+	// in order to complete the total CPU count to an even parity when using emulator thread isolation.
+	// Note: this feature is in Developer Preview.
+	// +optional
+	// +kubebuilder:default=false
+	// +default=false
```

Comment:
Don't we want it true by default?

Reply:
Since its an alpha "feature" in KubeVirt, IMO it should be explicitly enabled.

Reply:
Furthermore, users of this feature probably also set the `defaultRuntimeClass` field, so they need to edit the HCO manifest.
Both changes could be done in a single patch command.